code/modules/mob/living/living.dm

```diff
@@ -602,19 +602,24 @@ below 100 is not dizzy
 	if(!M.mind)
 		to_chat(M, "<span class='warning'>You don't have a mind.</span>")
 		return FALSE
-	if(!bypass && (key || ckey))
+	if(!bypass && client)
```

Comment:
These could be nested inside a single `if(!bypass)` branch to avoid repeated checks.